src/main/java/com/easyrest/pages/HomePage.java

```diff
@@ -1,4 +1,101 @@
 package com.easyrest.pages;
 
-public class HomePage {
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.awt.image.RescaleOp;
+
+public class HomePage extends BasePage{
+
+    private final By home = By.xpath("//*[text() = 'Home']");
```

Comment:
I think locators can be without final, because they are private - it is enough. If variable is final it should be written in capital letters.